consensus/broadcast/delayedBroadcast.go

```diff
@@ -35,16 +36,16 @@ type ArgsDelayedBlockBroadcaster struct {
 	AlarmScheduler        timersScheduler
 }
 
-type validatorHeaderBroadcastData struct {
-	headerHash           []byte
-	header               data.HeaderHandler
-	metaMiniBlocksData   map[uint32][]byte
-	metaTransactionsData map[string][][]byte
-	order                uint32
-	pkBytes              []byte
+type ValidatorHeaderBroadcastData struct {
```

Comment:
is there a reason why this struct and its fields were required to be exported?

Reply:
to be accessible in delayedBroadcasterMock